ydb/core/kqp/executer_actor/kqp_executer_impl.cpp

```diff
@@ -88,7 +88,7 @@ IActor* CreateKqpExecuter(IKqpGateway::TExecPhysicalRequest&& request, const TSt
 {
     if (request.Transactions.empty()) {
         // commit-only or rollback-only data transaction
-        YQL_ENSURE(request.LocksOp == ELocksOp::Commit || request.LocksOp == ELocksOp::Rollback);
+        AFL_ENSURE(request.LocksOp == ELocksOp::Commit || request.LocksOp == ELocksOp::Rollback)("real", (ui32)request.LocksOp);
```

Comment:
Kqp uses YQL_ENSURE. It only throws exception that will be caught and shown later.